piezo_web_app/PiezoWebApp/src/services/spark_job/validation/argument_validator.py

```diff
@@ -20,6 +22,16 @@ def validate(key, value, validation_rule):
     raise ValueError(f"Unexpected argument {key}")
 
 
+def _validate_label(value):
+    if not isinstance(value, str):
+        return ValidationResult(False, '"label" input must be a string', None)
+    if is_str_empty(value):
+        return ValidationResult(False, '"label" input cannot be empty', None)
+    if value.upper() == "ALL":
+        return ValidationResult(False, '"label" input cannot be "ALL"', None)
```

Comment:
If the user has input `"all"` then this message won't make complete sense. Add `(case insensitive)` to the end of the message.